.github/workflows/docker.yml

```
@@ -52,9 +54,7 @@ jobs:
           # Determine the platforms based on the branch
           PLATFORMS="linux/amd64"
           REF_NAME=$(echo "${GITHUB_REF#refs/*/}")
-          if [[ $GITHUB_REF == refs/tags/* ]]; then
-            PLATFORMS="linux/amd64,linux/arm64"
-          elif [ "$REF_NAME" = "production" ]; then
+          if [[ $GITHUB_REF == refs/tags/* || "$REF_NAME" = "$DEFAULT_BRANCH" ]]; then
```

what you have is fine, but this might work too, and would be a bit simpler:

```suggestion
          if [[ $GITHUB_REF == refs/tags/* || "$REF_NAME" = "${{ github.event.repository.default_branch }}" ]]; then
```